app/models/guide.rb

```diff
@@ -12,6 +12,7 @@ class Guide < ActiveRecord::Base
 
   before_validation on: :create do |object|
     object.content_id = SecureRandom.uuid
+    slug.try(:downcase!)
```

Comment:
Slug should never be nil. Effectively this is like doing `slug.downcase! rescue nil`.